src/index.html

```diff
@@ -5,7 +5,7 @@
   <title></title>
   <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500&display=swap" rel="stylesheet">
   <meta name="viewport" content="width=device-width, initial-scale=1" />
-  <script src="https://kit.fontawesome.com/3946063f04.js"></script>
+  <script src="https://kit.fontawesome.com/b86fe9836a.js"></script>
```

Comment:
@beejay1293 This won't be needed again when you use fontawesome the react way by having a library of icons that will be used in the entire project. Kindly refer to [https://github.com/FortAwesome/react-fontawesome](url) and `src/index` of the project on how it is done